modules/core/src/test/scala/org/tesselation/trust/DataGenerator.scala

```diff
@@ -0,0 +1,68 @@
+package org.tesselation.trust
+
+import cats.effect.{Async, Ref}
+import cats.syntax.all._
+
+import scala.util.Random
+
+import org.tesselation.infrastructure.trust.{TrustEdge, TrustNode}
+
+class DataGenerator[F[_]: Async] {
```

Comment:
I know it's just a test but looks like you don't need an Async here. Just `Ref.Make: Monad` would be enough :)

Reply:
Cats new version requires sync implicit on the ref signature. If I copy paste the code as is it doesn't work without it.